draft-ietf-quic-recovery.md

```diff
@@ -1202,15 +1205,25 @@ Invoked by loss detection from DetectLostPackets when new packets
 are detected lost.
 
 ~~~
+   InPersistentCongestion(oldest_loss_time):
+     elapsed_time = Now() - oldest_loss_time
+     pto = smoothed_rtt + 4 * rttvar + max_ack_delay
+     return elapsed_time > pto * (2 ^ kPersistentCongestionThreshold - 1)
+
    OnPacketsLost(lost_packets):
      // Remove lost packets from bytes_in_flight.
      for (lost_packet : lost_packets):
        bytes_in_flight -= lost_packet.size
+     smallest_lost_packet = lost_packets.first()
      largest_lost_packet = lost_packets.last()
```

Comment:
Slight tangent to this PR: what do folks think about changing these variables to `oldest_lost_packet` and `newest_lost_packet`?

Reply:
I don't have a strong preference either way.